src/users/users.resolver.ts

```
@@ -1,45 +1,34 @@
+    async getAllUsers(@Args('params') params: FindUserInput): Promise<User[] | void> {
+        return await this.userService.getAllUsers(params);
+    }
 
-  @Query(() => User, { name: 'user' })
-  async findUserById(
-    @Args('params') { _id }: FindUserInput,
-  ): Promise<User | void> {
-    return await this.userService.findUserById(_id);
-  }
+    @Query(() => User, { name: 'user' })
+    async findUserById(@Args('params') { _id }: FindUserInput): Promise<User | void> {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)